classes/class-object-sync-sf-wordpress.php

```diff
@@ -997,9 +997,10 @@ private function user_create( $params, $id_field = 'ID' ) {
 				do_action( $this->option_prefix . 'set_more_user_data', $user_id, $params, 'create' );
 
 				// Send notification of new user.
-				// todo: Figure out what permissions ought to get notifications for this and make sure it works the right way.
-				wp_new_user_notification( $user_id, null, 'both' );
-
+				// todo: Figure out what permissions ought to get notifications for this and make sure it works the right way. In the meantime, allow developers to prevent the notification from being sent.
+				if ( apply_filters( $this->option_prefix . 'send_new_user_notification', true, $user_id, $params ) ) {
+					wp_new_user_notification( $user_id, null, 'both' );
+				}
```

Comment:
It is possible to user the default WP hooks `wp_send_new_user_notification_to_admin` and `wp_send_new_user_notification_to_user` but the problem with this is that by the time those hooks run, the developer cannot know where the user came from.

This is particularly bad when doing an "import" of existing contacts into wp ... perhaps you want to import users but not let them know about it.